BogdanShemetov.md

```diff
@@ -16,26 +16,28 @@ We plan on using Arduino Nano as a brain for a display (to graph and calculate d
-| Protoboard      | https://1wire.com.ua/ru/maketnaja-plata-90x150-mm.html                                 | 2 pc    | $2,33   |
-| Breadboard      | https://1wire.com.ua/ru/-maketnaja-plata-830.html                                      | 1 pc    | $2,37   |
-| LCD Display     | https://diylab.com.ua/ua/p1176983248-tft-lcd-spi.html                                  | 1 pc    | $17,69  |
-| Mosfet          | https://diylab.com.ua/ua/p704623085-silovij-modul-mosfet.html                          | 1 pc    | $2,09   |
-| Resistors 1K    | https://diylab.com.ua/ua/p131590157-rezistori-1kom-025vt.html                          | 2 pc    | $0,13   |
-| Keypad          | https://diylab.com.ua/ua/p706783746-matrichna-klaviatura-4x4.html                      | 1 pc    | $3,44   |
-| Delivery        | NovaPoshta                                                                             | 3 pc    | $2,03   |
-| Casing (owned)  | Owned                                                                                  | 1 pc    | $0      |
-| Total           |                                                                                        |         | $138.92 |
+| Product               | Supplier/Link                                                                          | Quanity | Cost    |
+| --------------------- | ---------------------------------------------------------------------------------------| ------- | ------- |
+| pH Meter              | https://arduino.ua/prod3490-gravity-analogovii-datchikizmeritel-ph-v2-ot-dfrobot 	     | 1 pc    | $31,24  |
+| Mini USB Cable        | https://arduino.ua/prod892-usb-2-0-kabel-am-miniusb-0-8m-atcom-s-ferritom              | 1 pc    | $1,19   |
+| Valve                 | https://arduino.ua/prod3109-mini-klapan-elektromagnitnii-12v-trehhodovoi               | 1 pc    | $3,76   |
+| Voltage shifter       | https://arduino.ua/prod2165-preobrazovatel-logicheskih-yrovnei-dvynapravlennii         | 1 pc    | $1,11   |
+| 12V Step-up           | https://arduino.ua/prod5033-povishaushhii-dc-dc-preobrazovatel-12v-u3v40f12-ot-pololu  | 1 pc    | $2,81   |
+| Jumpers               | https://arduino.ua/prod1596-komplekt-peremichek-arduino-120sht                         | 2 pc    | $2,68   |
+| Soldering Kit         | https://arduino.ua/prod3398-nabor-dlya-paiki-elektroniki-s-diagonalnimi-kysachkami     | 1 pc    | $26,43  |
+| Wire 12AWG            | https://arduino.ua/prod889-kabel-12awg-v-silikonovoi-izolyacii-50sm-chernii            | 1 pc    | $1,35   |
+| Power Supply          | https://arduino.ua/prod3293-blok-pitaniya-5v-3a-usb-type-c-dlya-raspberry-pi-4         | 1 pc    | $7,00   |
+| USB C                 | https://arduino.ua/prod2783-plata-perehodnik-usb-type-c-na-dip-2-54                    | 1 pc    | $0,32   |
+| Calibration solutions | https://diylab.com.ua/ua/p706783746-matrichna-klaviatura-4x4.html                      | 1 pc    | 5,68    |
```

Comment:
hey @BogdanShemetov the link goes to a matrix keyboard instead of calibration solution?